backends/gdx-backends-gwt/src/com/badlogic/gdx/backends/gwt/GwtApplication.java

```diff
@@ -92,7 +92,10 @@ public abstract class GwtApplication implements EntryPoint, Application {
 	public abstract GwtApplicationConfiguration getConfig ();
 
 	public String getPreloaderBaseURL () {
-		return GWT.getHostPageBaseURL() + "assets/";
+		String moduleUrl = GWT.getModuleBaseURL();
+		// Total Length - len("html") - len("/")
```

Comment:
Please add a comment here what happens exactly, it is hard to understand and "correctLength" variable name does not help here (what is "correct" in this context and why is the returned path not correct?) Somethink like "find asset path next to html path"